charts/kube-plex/templates/deployment.yaml

```diff
@@ -133,6 +133,32 @@ spec:
         - name: "NO_PROXY"
           value: "{{.Values.proxy.noproxy}}"
   {{- end }}
+{{- end }}
+# Official kube-plex container environment variables
+{{- if .Values.advertiseIp }}
+        - name: "ADVERTISE_IP"
+          value: "{{.Values.advertiseIp}}"
+{{- end }}
+{{- if .Values.changeConfigDirOwnership }}
+        - name: "CHANGE_CONFIG_DIR_OWNERSHIP"
+          value: "{{.Values.changeConfigDirOwnership}}"
+{{- end }}
+{{- if .Values.allowedNetworks }}
+        - name: "ALLOWED_NETWORKS"
+          value: "{{include "joinListWithComma" .Values.allowedNetworks}}"
```

Comment:
I think it may be simpler to just make this `{{ .Values.allowedNetworks }}` and require the user to specify a comma separated list, but it's not a particular big deal either way 😄 (although it makes things a little simpler and reduces the Helm templating magic, which could well come back to bite 😅)